coreos-ostree-importer/send-ostree-import-request.py

```diff
@@ -53,26 +57,24 @@ def parse_args():
 
 
 def send_ostree_import_request(args):
-    buildid = args.build
-    build = GenericBuildMeta(build=buildid)
+    build = GenericBuildMeta(build=args.build, basearch=args.arch)
```

Comment:
This isn't new, though I think here we should have e.g.

```python
buildid = build['buildid']
```

and use that below. Because it doesn't seem appropriate in a signing context to just pass through `latest` unresolved if `--build` wasn't actually passed. We should probably also just remove that default from the argparse argument.

Anyway, since it's not new, OK to defer.

Reply:
ok tried to address this in a new commit - pushed